dallinger/recruiters.py

```diff
@@ -77,6 +81,11 @@ class PsiTurkRecruiter(Recruiter):
 
     def __init__(self):
         """Set up the connection to MTurk and psiTurk web services."""
+
+        # commit to the db so if anything goes wrong the rest of the
+        # request is safe
+        session.commit()
```

Comment:
@thomasmorgan as you describe it, there must be code elsewhere that creates or modifies objects in the database and then instantiates a recruiter (and thus calls this code). can we put the commits directly after the objects are created or modified?